### PR TITLE
feat: updated units and added unit multiplier

### PR DESCRIPTION
**Pull request by Ryoken**
#### Any background context you want to provide?
see ticket

#### What's this PR do?
Changes the units for one BETTER import and adds the ability to scale units before adding them to extra_data.

#### How should this be manually tested?
Run a BETTER analysis and check that GHG Emissions Reduction is reported in MTCO2e and matches that in the HTML report.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2843

#### Screenshots (if appropriate)
Screen Shot 2022-03-17 at 12 48 27 AM

Screen Shot 2022-03-17 at 12 48 34 AM

### CODE REVIEW

File: seed/analysis_pipelines/better/pipeline.py [hunk 1]
```diff
@@ -368,69 +369,82 @@ def _process_results(self, analysis_id):
...
         ),
         ExtraDataColumnPath(
             'better_energy_savings_fuel',
             'BETTER Potential Fuel Energy Savings (kWh)',
+            1,
             'assessment.assessment_energy_use.energy_savings_f'
         ),
         ExtraDataColumnPath(
             'better_ghg_reductions_electricity',
             'BETTER Potential Electricity GHG Emissions Reduction (kgCO2e)',
```
Comment (nllong) [inline seed/analysis_pipelines/better/pipeline.py:426]: we'll need to convert this to tons co2e, but we can do that in another PR.

Comment (nllong) [reply seed/analysis_pipelines/better/pipeline.py:426]: Actually, should that happen in this PR, since you just created the multiplier argument?

Comment (Ryoken) [reply seed/analysis_pipelines/better/pipeline.py]: It should, yes.  Updating!

File: seed/analysis_pipelines/better/pipeline.py [hunk 2]
```diff
@@ -368,69 +369,82 @@ def _process_results(self, analysis_id):
...
         ExtraDataColumnPath(
             'better_energy_savings_combined',
             'BETTER Potential Energy Savings (kWh)',
+            1,
             'assessment.assessment_energy_use.energy_savings_combined'
         ),
         ExtraDataColumnPath(
             'better_ghg_reductions_combined',
-            'BETTER Potential GHG Emissions Reduction (kgCO2e)',
+            'BETTER Potential GHG Emissions Reduction (MTCO2e)',
```
Comment (nllong) [inline seed/analysis_pipelines/better/pipeline.py:383]: We are just using `t` for metric ton. That is what AT is using.

```suggestion
            'BETTER Potential GHG Emissions Reduction (t CO2e/yr)',
```

File: seed/analysis_pipelines/better/pipeline.py [hunk 3]
```diff
@@ -368,69 +369,82 @@ def _process_results(self, analysis_id):
...
         ExtraDataColumnPath(
             'better_energy_savings_fuel',
             'BETTER Potential Fuel Energy Savings (kWh)',
+            1,
             'assessment.assessment_energy_use.energy_savings_f'
         ),
         ExtraDataColumnPath(
             'better_ghg_reductions_electricity',
-            'BETTER Potential Electricity GHG Emissions Reduction (kgCO2e)',
+            'BETTER Potential Electricity GHG Emissions Reduction (MTCO2e)',
```
Comment (nllong) [inline seed/analysis_pipelines/better/pipeline.py:426]:
```suggestion
            'BETTER Potential Electricity GHG Emissions Reduction (t CO2e/yr)',
```

File: seed/analysis_pipelines/better/pipeline.py [hunk 4]
```diff
@@ -368,69 +369,82 @@ def _process_results(self, analysis_id):
...
+            .001,
             'assessment.assessment_energy_use.ghg_reductions_e'
         ),
         ExtraDataColumnPath(
             'better_ghg_reductions_fuel',
-            'BETTER Potential Fuel GHG Emissions Reduction (kgCO2e)',
+            'BETTER Potential Fuel GHG Emissions Reduction (MTCO2e)',
```
Comment (nllong) [inline seed/analysis_pipelines/better/pipeline.py:432]:
```suggestion
            'BETTER Potential Fuel GHG Emissions Reduction (t CO2e/yr)',
```

**Review by nllong — APPROVED** [review]
looks good, approving now. Merge after accepting suggestions.
